tsconfig.json

```diff
@@ -9,7 +9,7 @@
         "sourceMap": true,
         "rootDir": ".",
         "noImplicitReturns": true,
-        "strictNullChecks": true
+        "strictNullChecks": false
```

Comment:
I agree with you. I also prefer strictNullChecks. Problem is Throttler class in async.ts. ts compiler will generate errors when strictNullChecks.
I copied Throttler from vscode sources. I don't like to change it to be compatible with strictNullChecks. It would be best to turn off strictNullChecks per file / function but I do not know how to do it.
Any suggestion how to handle it without changing Throttler code?